README.md

```diff
@@ -1,15 +1,15 @@
 # expressly
-Express-style router for Fastly's Compute@Edge.
+Express-style router for Fastly Compute.
 
 ## Using expressly
 
 [Check out the docs](https://expressly.edgecompute.app/).
 
 ---
 
-First, head over to [developer.fastly.com](https://developer.fastly.com) to get started with JavaScript on Fastly's Compute@Edge:
+First, head over to [developer.fastly.com](https://developer.fastly.com) to get started with JavaScript on Fastly Compute:
 
-1. Learn about Compute@Edge: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
+1. Learn about Compute: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
```

Comment:
I feel like this could use the "Fastly" prefix...

```suggestion
1. Learn about Fastly Compute: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
```